.github/workflows/lint.yml

```diff
@@ -1,4 +1,4 @@
-on: [push, pull_request]
+on: push
```

Comment:
can we still run the w3c tests when the pull_request is local?

Reply:
Well, if the pull request is local, it's a push, right?
That's actually the setting you have in `validate-repos`.

Reply:
Ha, makes perfect sense indeed :)